geonode_mapstore_client/client/js/components/home/DetailsPanel.jsx

```diff
@@ -95,10 +95,12 @@ function DetailsPanel({
     editAbstract,
     editThumbnail,
     activeEditMode,
-    closePanel
+    closePanel,
+    favourite,
+    onFavourite,
+    isLogged
```

Comment:
I think we should use something more related to favourite instead of isLogged. isLogged is related to the user and the DetailsPanel component should not be aware of the user state

maybe `enableFavourite`?